templates/tips/edit_tip.html

```diff
@@ -1,11 +1,33 @@
-{% extends "base.html" %} {% load material_form %}{%load static%} {%block content%}
-<div class="editfrm">
-	<p class="edittip"><h1>Edit tip</h1></p>
-	<form method="POST">
-		{% csrf_token %}
-		{% form form=form %}{% endform %}
-		<input type="submit" value="Post">
-	</form>
+{% extends "base.html" %}
+{% load crispy_forms_filters %} {% load static %}
+{% load crispy_forms_tags %}
+{% block content %}
+{% if user.is_authenticated %}
+{% if user.get_session_auth_hash == tip.user_token %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/accounts/edit_tip.css' %}" />
+<div id="edit_tip">
+	<div class="d-flex justify-content-center h-100">
+		<div class="card">
+			<div class="card-header">
+				<h3>Edit Tip</h3>
+			</div>
+			<div class="card-body" style="color: rgb(0, 0, 0); background: rgba(252, 252, 252, 0.3); width: 500px;">
+				<form method="post">
+					{% csrf_token %}
+					{{ form|crispy }}
+					<div class="form-group">
+						<button class="btn btn-primary mt-3" type="submit">Edit Tip</button>
+						<a class="btn btn-primary mt-3" href="/tips/">Cancel</a>
+					</div>
+				</form>
+			</div>
+		</div>
+	</div>
 </div>
-<a class="btn btn-primary" href="/tips/" role="button">Cancel</a>
-{% endblock %}
+{% else %}
+<div>Premision Denied!</div>
```

Comment:
I get permission denied to edit, or delete on a tip I added 

![image](https://user-images.githubusercontent.com/34008861/170834716-0a45c7c7-b381-4177-a9a7-10930532d34b.png)

Reply:
Fixed